docs/pages/repo/docs/reference/system-variables.mdx

```diff
@@ -0,0 +1,28 @@
+---
+title: System environment variables
+description: System environment variables are built-in variables that are available to all scripts.
+---
+
+## System environment variables
+
+By setting certain environment variables, you can change Turborepo's behavior. This can be useful for creating specific configurations for different environments and machines.
+
+| Variable                           | Description                                                                                                                                                                                                                                   |
+| ---------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `TURBO_API`                        | Set the base URL for [Remote Cache](/repo/docs/core-concepts/remote-caching).                                                                                                                                                                 |
+| `TURBO_BINARY_PATH`                | Manually set the path to the `turbo` binary.                                                                                                                                                                                                  |
+| `TURBO_CI_VENDOR_ENV_KEY`          | Set a prefix for environment variables that you want **excluded** from [Framework Inference](/repo/docs/core-concepts/caching/environment-variable-inputs#framework-inference).                                                               |
+| `TURBO_FORCE`                      | Always force all tasks in your pipelines to run in full, opting out of all caching.                                                                                                                                                           |
+| `TURBO_HASH`                       | The hash of the currently running task.                                                                                                                                                                                                       |
```

Comment:
I'm not sure if this belongs here. This env var doesn't control `turbo`'s behavior, but `turbo` provides it to tasks during their execution.